src/axom/core/docs/sphinx/core_containers.rst

```diff
@@ -0,0 +1,13 @@
+.. ## Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+******************************************************
+Core containers
+******************************************************
+
+Axom Core contains the Array and StackArray classes.  Among other things, these
+data containers facilitate porting code that uses `std::vector` to the GPU.
```

Comment:
So this be more to the point; e.g., these containers can be used to replace STL containers, such as std::vector, that are incompatible with GPU code?

Reply:
Yes.  I still need to talk about core containers.

Reply:
done.

Reply:
@rhornung67 , will you please take another look at this?